removed the empty space on the right of the header and shifted buttons

vladutvoicu opened the pull request:
**Notes for Reviewers**

<!--tag the issue id  -->
This PR fixes #129 

Describe the changes you've made:

removed the empty space on the right of the header and shifted buttons (changes made to "src/Components/Alternative/Header.module.css")

Screenshots for the change:

![2c5251724a69638f35dd0b685b757ab9](https://user-images.githubusercontent.com/107242590/230161622-6498065e-80cc-4d9c-bd8d-dac420fa274d.png)


<!--
Thank you for contributing to GDSCITM! 

Contributing Conventions:
- Include descriptive PR titles with [<component-name>] prepended.


By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

ArunPatel02 reviewed (changes requested):
@vladutvoicu  could you please  decreases the space between two navigation menu and instead of justify-center : space-evenly can you implement justify-end and give margin from right.

![image](https://user-images.githubusercontent.com/89798093/230282991-608ab664-b052-45e2-8002-30083af6eab2.png)

Yashsharma1911 commented:
some good reviews here @vladutvoicu from @ArunPatel02 do you need any help?

vladutvoicu commented:
> some good reviews here @vladutvoicu from @ArunPatel02 do you need any help?

Working on it right now

vladutvoicu commented:
I changed justify-content to "flex-end" and added margin to the right side

![e86a71e0d8300ea154294b283eabd55a (1)](https://user-images.githubusercontent.com/107242590/230348262-2d01ee3c-d65c-4a83-a326-273db86ba1c3.png)

ArunPatel02 commented:
> I changed justify-content to "flex-end" and added margin to the right side
> 
> ![e86a71e0d8300ea154294b283eabd55a (1)](https://user-images.githubusercontent.com/107242590/230348262-2d01ee3c-d65c-4a83-a326-273db86ba1c3.png)

great work @vladutvoicu  thanks a lot for your contribution
